src/components/ColorPicker/ColorPickerItem/ColorPickerItem.tsx

```diff
@@ -2,11 +2,19 @@ import * as React from 'react';
 import { st, classes } from './ColorPickerItem.st.css';
 import { RadioButton } from 'wix-ui-core/radio-button';
 import { RadioButtonProps } from 'wix-ui-core/dist/src/components/radio-button/RadioButton';
-import { colorPickerItemDataHook } from '../dataHooks';
+import {
+  colorPickerItemDataHook,
+  colorPickerItemTooltipDataHook,
+} from '../dataHooks';
+import { Tooltip } from '../../Tooltip';
+import { MobileTooltip } from './MobileTooltip';
+import { TPAComponentsConsumer } from '../../TPAComponentsConfig';
 
 export interface ColorPickerItemProps extends RadioButtonProps {
   className?: string;
   key?: number;
+  disabled?: boolean;
```

Comment:
After talking to @johannaasseraf I understand that the item isn't actually disabled.
This means we should probably change the name of the prop to `unavailable` because we still want the item to be selectable.
We might add a `disabled` prop in the future, which will really set the input to be `disabled` and the item won't be selectable.

You can then change the stylable state to `unavailable` as well

Reply:
After the very last discussion we decided that we currently need `disabled` state (cannot be selected, but with tooltip). In future we plan to have addition `unavailable` state (or we can call it `crossedOut`) which will be selectable but with crossed out style.

Reply:
@johannaasseraf Please review.

Reply:
I just didn't really want the disabled state to be connected to the crossed-out/unavailable design. If you want it can be a separate prop that is turned on for our case. Makes sense?

Reply:
Absolutely. So to recap. Two props: `disabled` and `isCrossedOut`. All good?

Reply:
Makes sense to me.. What do you say @jonathanadler ?

Reply:
`unavailable ` or `isCrossedOut`  -- both variants are good in my opinion. Don't know which one to pick out.

Reply:
I'm ok with crossedout, sounds less specific for a certain vertical

Reply:
yeah lets go with the crossed-out one 👍